### PR TITLE
[4.0] New clean layout » separate sections

### PR DESCRIPTION
**Pull request by HRIT-Florian**
### Summary of Changes
Hey guys, the backend design needs a small but effective change.

See yourself

Before:
before2

After:
after2

--

Before:
before

After:
after

--

Before:
grafik

After:
grafik

If you like what you see, we can adapt these easy in all views next.
So, what about to take the PR ? ... next

### CODE REVIEW

**Comment by brianteeman** [comment]
LGTM

**Comment by Quy** [comment]
See red arrows where they don't look good.

**Comment by HRIT-Florian** [comment]
@Quy maybe, but it's not part of that PR and its same as before.
Anyway I think the new layout is a lot of smarter, takes more contrast beetween the sections and  makes more use of the width on smaller devices.

Your Screenshot includes also not all changes, because it looks like this after the change:

After:
grafik

Before:
grafik

We have a straight left edge of the containers.
Please check with all style changes.

**Comment by infograf768** [comment]
I have unwanted consequences here when not using card but only `<div class="col-md-12">`

### Before patch

Screen Shot 2020-06-09 at 11 05 01

### After patch
Screen Shot 2020-06-09 at 11 11 01
Screen Shot 2020-06-09 at 11 05 41

**Comment by HRIT-Florian** [comment]
@infograf768 yes, I know these, and it expects anyway a new pr to change the classes in the views.
Here we hefe different layouts, its anyway bad and have to be fixed anyway.

For example, after my changes, the cofig permissions looks like these:

grafik

So the question is, why we have anyway different layouts?
We should change all layouts to a uniform standard.
For these my pr could also help to detect the differences..
Next we can change the layouts which do not follow the standard.

**Comment by infograf768** [comment]
We can't expect a next step imho. Such a pr should deal with all situations.

**Comment by HRIT-Florian** [comment]
Okay, I will check any view and will add the changes if needed in these PR nex...

**Comment by HRIT-Florian** [comment]
@infograf768 which view of which component is the view in your screenshot?
I've checked now any view in current 4.0-dev branche. And it looks really fine, except for the view shown by you, which is unknown to me? Let me know and I will fix it. Finally I think, the PR is acceptable, because minor adjustments can still be made!

**Comment by uglyeoin** [comment]
@infograf768 please can you answer @Didldu-Florian so we can test it as part of PBF

**Comment by priyankaSahutekdi** [comment]
I have tested this item :white_check_mark: successfully on 1fac65706903d7e587ec91c411f51f83af563a03<br /><br />The design is looking so clean now<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29514">issues.joomla.org/tracker/joomla-cms/29514</a>.</sub>

**Comment by priyankaSahutekdi** [comment]
I have not tested this item.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29514">issues.joomla.org/tracker/joomla-cms/29514</a>.</sub>

**Comment by gostn** [comment]
@priyankaSahutekdi you shouldn´t test a pr having conflicting files.

**Comment by HRIT-Florian** [comment]
@Quy , options-form legend border is now also added by https://github.com/joomla/joomla-cms/pull/29514/commits/0fa941c4ad03e2254eb3e84713d5f08ea40486e9

grafik

Final I don't know what all these discussions are about, because the design improves a lot of things. 
And everything else can easily be followed up.

**Comment by HRIT-Florian** [comment]
Can someone remove "Conflicting Files" Label? Thx

**Comment by Quy** [comment]
I find the highlighted/active tab distracting. For the side area, should it have a white background?

**Comment by HRIT-Florian** [comment]
@Quy maybe better now:
grafik

Sorry for such a lot mimimis, there are so many things to do with that new design better as with the old one.
Finally for me it looks much more cleaner as the white striped style! So let it get into core next bevore the PR is fucked up! The mentioned things for me all right, of courese, but these are only small improvements, which can also be criticised in the current Design. I'm sure, any small criticised could be easy fixed after that also!

File: api/language/en-GB/joomla.ini [hunk 1]
```diff
@@ -279,6 +279,7 @@ JFIELD_TITLE_DESC="Title"
 JFIELD_VERSION_HISTORY_DESC="This button allows you to open a window to view older versions of this item."
 JFIELD_VERSION_HISTORY_LABEL="Prior Versions"
 JFIELD_VERSION_HISTORY_SELECT="View Prior Versions"
+JGLOBAL="Global"
```
Comment (infograf768) [inline api/language/en-GB/joomla.ini]: Needs to be added to joomla.ini backend ini file
Screen Shot 2020-11-11 at 07 41 08

Also, "Global" has not much sense imho as all parameters there are not global but specific for the item.

Comment (HRIT-Florian) [reply api/language/en-GB/joomla.ini]: @infograf768 its moved first. I can change the name if you have any better suggestion?

**Comment by chmst** [comment]
Looks good - but I think that the legend "Global" is not good. I know that it is the fieldset name - and it is called so from developers who use these params everywhere. But what means "global" for users? @infograf768 , @brianteeman?

**Comment by brianteeman** [comment]
Without looking at the specific reference for this so that I can comment on the word only

Global is for the entire site. 
eg Global Configuration
Use Global
etc

**Comment by chmst** [comment]
global

Maybe "Status" is better?

**Comment by brianteeman** [comment]
Now that I look at it - global is definitely wrong
Does it even need a text?

**Comment by chmst** [comment]
No, it doesn't. @Didldu-Florian can you remove the legend? <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29514">issues.joomla.org/tracker/joomla-cms/29514</a>.</sub>

**Comment by HRIT-Florian** [comment]
@chmst I can, the only problem is that afterwards we are not in line and have to push manually with px
grafik

I will suggest, using only fielset options-forms with legends, so we should find a good title, maybe?

**Comment by chmst** [comment]
So we need a good name for a quick solution. Controls?  

On the other Hand - do we need the legend "Article Text" around the the text input? Isn't it self explaining also without legend?

**Comment by HRIT-Florian** [comment]
For me "Controls" sounds good. 
Of course we could also remove the other legend, but then you can basically think about removing some legends everywhere.

**Comment by infograf768** [comment]
Basic Parameters ?

**Comment by chmst** [comment]
Better than "Global". Let's take this.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29514">issues.joomla.org/tracker/joomla-cms/29514</a>.</sub>

**Comment by brianteeman** [comment]
For me the word parameters is wrong - but what does that matter

**Comment by ceford** [comment]
Article Settings?

I revised the Help screens for forms in recent months, many of them without legends. They are mostly referred to as Left and Right Panel. I would like to see the legends kept!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29514">issues.joomla.org/tracker/joomla-cms/29514</a>.</sub>

**Comment by HRIT-Florian** [comment]
@ceford I think the same, we need uniform legends for all field sets.
But finnaly not in these PR, next... so it's a never ending story

**Comment by Quy** [comment]
Closing in favor of #33100. Thanks.

**Pull request closed without merging**